acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -474,6 +532,23 @@ def _create_model_cos_zenith_binned(self,
                 livetime_per_obs.append(obs.observation_live_time_duration)
             bin_center.append(np.sum([wcos.value for wcos in weighted_cos_zenith_bin_per_obs]) / np.sum([livet.value for livet in livetime_per_obs]))
 
+        if self.verbose:
+            print("cos zenith bin edges: ", list(np.round(cos_zenith_bin, 2)))
+            print("cos zenith bin centers: ", list(np.round(bin_center, 2)))
+            print(f"observation per bin: ", list(np.histogram(cos_zenith_observations, bins=cos_zenith_bin)[0]))
+            print(f"livetime per bin [s]: ", list(
+                np.histogram(cos_zenith_observations, bins=cos_zenith_bin, weights=livetime_observations)[0].astype(int)))
+            if per_wobble:
+                wobble_observations_bool_arr = [(np.array(wobble_observations.tolist()) == wobble) for wobble in
+                                                np.unique(np.array(wobble_observations))]
+                livetime_observations_and_wobble = [np.array(livetime_observations) * wobble_bool for wobble_bool in
+                                                    wobble_observations_bool_arr]
+                for i, wobble in enumerate(np.unique(np.array(wobble_observations))):
+                    print(
+                        f"{wobble} observation per bin: {list(np.histogram(cos_zenith_observations, bins=cos_zenith_bin, weights=1 * wobble_observations_bool_arr[i])[0])}")
+                    print(
+                        f"{wobble} livetime per bin: {list(np.histogram(cos_zenith_observations, bins=cos_zenith_bin, weights=livetime_observations_and_wobble[i])[0].astype(int))}")
+
```

Comment:
I would suggest to use logging.info instead of print as it allow more control at library level, @gabemery what is you opinion on this ?

Reply:
I propose an overall rework of the logging. Including removing of the prints. Since such a PR will be needed we can ignore it here and change it later. Next implementation would probably not include 'verbose' and use only logging at specific levels.